common/src/main/java/tds/common/cache/NameAwareGuavaCacheManager.java

```diff
@@ -41,6 +41,9 @@ protected Cache createGuavaCache(final String name) {
             case CacheType.LONG_TERM:
                 cacheBuilder.expireAfterWrite(cacheProperties.getExpireTimeLong(), SECONDS);
                 break;
+            default:
+                cacheBuilder.expireAfterWrite(cacheProperties.getExpireTimeDefault(), SECONDS);
+                break;
```

Comment:
Added a default cache expiration to our Guava cache that will kick anything out after 24 hours.